shoop_tests/front/test_customer_information.py

```diff
@@ -52,3 +52,15 @@ def test_new_user_information_edit():
 
     assert response.status_code == 302
     assert get_user_model().objects.get(pk=user.pk).email == new_email
+
+
+@pytest.mark.django_db
+def test_user_information_unavailable_if_not_logged_in():
+    get_default_shop()  # Front middleware needs a Shop to exists
+    client = SmartClient()
+    customer_edit_url = reverse("shoop:customer_edit")
+    login_path = reverse("shoop:login")
```

Comment:
This should probably be `resolve_url(settings.LOGIN_URL)`, actually ([that's what Django does](https://github.com/django/django/blob/1.8.4/django/contrib/auth/decorators.py#L24))